addons/slideshow/functions/fnc_autoTransition.sqf

```diff
@@ -34,6 +35,9 @@ missionNamespace setVariable [_varString, _currentSlide];
     _x setObjectTextureGlobal [0, _images select _currentSlide];
 } count _objects;
 
+//Raise event upon slide change
```

Comment:
```suggestion
// Raise event upon slide change
```
A bit of a redundant comment though.

Reply:
Good point. removed it : 44498067e6a7654d944b4f4a52b75ebd91c97efb